olive/__init__.py

```diff
@@ -14,4 +14,4 @@
 _logger.addHandler(_sc)
 _logger.propagate = False
 
-__version__ = "0.5.0"
+__version__ = "0.5.1"
```

Comment:
should it be 0.5.1 or 0.6.0? So far we have incremented the second number and did patch release for the third.

Reply:
I prefer 0.6.0.

Reply:
yeah i can do 0.6.0